packages/client/src/index.test.ts

```diff
@@ -34,19 +34,18 @@ describe('init client', () => {
   beforeAll((done) => {
     client = new LogtoClient(
       {
-        discoveryUrl: 'https://logto.dev/oidc/.well-known/openid-configuration',
+        logtoUrl: 'https://logto.dev',
         clientId: 'foo',
-        redirectUris: ['http://localhost:3000/callback'],
       },
       done
     );
   });
   test('openid configuration', async () => {
-    const configuration = client.issuer.metadata;
-    expect(configuration.authorization_endpoint).toContain('oidc/auth');
+    const configuration = client.issuer?.metadata;
+    expect(configuration?.authorization_endpoint).toContain('oidc/auth');
   });
   test('get login url and codeVerifier', () => {
-    const [url, codeVerifier] = client.getLoginUrlAndCodeVerifier();
+    const [url, codeVerifier] = client.getLoginUrlAndCodeVerifier('http://localhost:3000/callback');
     console.log(url);
     console.log(codeVerifier);
     expect(typeof url).toEqual('string');
```

Comment:
since we have defined the client.getLoginUrlAndCodeVerifier's return type as [string, string]. Feels like no need to validate within ut case here?

Reply:
yep, useless validation here, just for the `console.log`